### DISCUSSION TITLE
Floating windows acting crazy, either flickering like mad or flickering and following the cursor

### DISCUSSION THREAD

**Discussion by ghost** [i3/i3#4666] [Q&A]
So far, I've seen this happen with Firefox's privacy popup (only when it pops up on its own), and with GIMP's color editor.

Behavior survives a reboot, and survives reloading or restarting i3.  This doesn't happen with *all* floating windows, just some.

I think something in my config is causing it, but I'm not sure.
I even tried commenting out the lines that say
```
default_floating_border normal
```
and
```
for_window [floating] border normal
```
but that didn't help

Note: The flickering is much faster than it appears in the video.
Note2: Config attached below
Note3: I've also tried killing picom
Note4: This is i3-gaps 4.19.1 on Manjaro 21.1.16 (i3 community edition)
Note5: `inxi -Fxxxza` output attached

[i3-config.txt](https://github.com/i3/i3/files/7506840/i3-config.txt)

[inxi.txt](https://github.com/i3/i3/files/7507387/inxi.txt)

https://user-images.githubusercontent.com/38673813/140975151-5d7e51cc-17ca-45a7-8e06-7a0db9ffc530.mp4

**Comment by Airblader** [comment]
This is very unspecific, but my first guess would be the compositor. Does it also happen if the compositor isn't running?

**Reply by ghost** [reply]
I've run it with and without `picom` running.

I will try to compare it to another machine and see if I get the same behavior, and whether or not I can find a substantive difference in the configs.

I've already commented out the only recent config change I can recall making.

Thanks.

**Reply by ghost** [reply]
Just did an sdiff between that box's config and my Pinebook Pro (also running Manjaro i3 community edition).
I don't see any substantive differences :/

If you have a recommended default config, I can try running that and attempting to reproduce the problem.

Thanks again, airblader.

**Reply by Airblader** [reply]
I don't think it's an i3 configuration problem. This is more likely to either be compositing or graphics driver related.

**Reply by ghost** [reply]
Interesting. I don't know what could be moving the mouse so strangely like that.

I took a better quality screencap to give a better idea how it's behaving. I also killed picom and xbanish (slim chance it's having any effect, but I had to check)

The odd thing is that the window starts drifting to the left. I'll try it again using a blank/stock config a little later and see if I can still reproduce it.

I also tried messing with my dual-monitor layout in arandr, but that didn't have any effect.

Thanks.

https://user-images.githubusercontent.com/38673813/141173350-e51f49db-c205-4572-8983-af7a690664eb.mp4

**Reply by ghost** [reply]
Well, I did a reboot to troubleshoot further. I could have sworn I got it to happen again, but now it doesn't want to do it at all... but it has done it (or something very similar) before.

I guess the trail is cold now. Thanks for your time.
